userguide/jails.rst

```diff
@@ -663,14 +663,22 @@ example is shown in :numref:`Figure %s <jail_overview_fig>`.
    +-------------------+----------------------------------------------------+
 
 
+Select multiple jails from the list by setting the leftmost checkboxes.
```

Comment:
This is an imperative, but really should mention that it is possible instead.  And "leftmost" is a rare word:
"Operations can be applied to multiple jails by selecting those jails with the checkboxes on the left."